CHANGELOG.md

```diff
@@ -40,9 +40,10 @@
 - `resourcetotelemetry`: Ensure resource attributes are added to summary
   and exponential histogram data points. (#7523)
 
-## Deprecations
+## 🚩 Deprecations 🚩
 
 - Deprecated otel_to_hec_fields.name setting from splunkhec exporter. (#7560)
+- Deprecated log_names setting from filter processor. (#7552)
```

Comment:
I'm blocking this for a few moments, as this is now outdated due to the new release.

Reply:
Updated now. Rebased from main.